fem/tmop_tools.hpp

```diff
@@ -224,14 +226,26 @@ class TMOPNewtonSolver : public LBFGSSolver
    /// (ii) surface fitting weight.
    virtual void ProcessNewState(const Vector &x) const;
 
-   /** @name Methods for adaptive surface fitting weight. (Experimental) */
+   /** @name Methods for adaptive surface fitting. These control the behavior of
+       the weight and the termination of the solver. (Experimental) */
```

Comment:
Don't you need `///@{  ... ///@}` to actually group the functions in the doxygen.